### ISSUE TITLE
step 48 build a calorie counter project not liking the label for in the temperate literal codel

### ISSUE THREAD

**Issue by blindgeekzone** [freeCodeCamp/freeCodeCamp#62341]
### Describe the Issue

hi. a blind screen reader user, i then add the <label for in the step 48 of the build a calorie counter project, says the for label and the text is not being recognized in the fcc editor. and have tried multiple browsers and also looked in my vs code local copy and then typed up the line of code a few times, not passing, as fcc is not behaving properly. and then giving me like in chrome 0 of 103 steps. so something is wrong, please fix the issue.,

### Affected Page

https://www.freecodecamp.org/learn/full-stack-developer/workshop-calorie-counter/step-48

### Steps to Reproduce

hi,  i then type the code in this in the editor. will paste from vs code as got a local copy on my hard drive.   const HTMLString = `<label for="${entryDropdown.value}-${entryNumber}-name">Entry ${entryNumber} Name</label>`;
. Go to '...'
2. Click on '...'
3. Scroll down to '...'

### Expected behavior

that it should pass the step and if not, then tell me the tests as it is not in firefox, and then tell me it passed the step, so this is the third bug report reporting.

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

hi, cannot do screen shots, as using a screen reader. but please fix this issue, unless something is wrong with my code. the fcc editor is not behaving correctly, and this is the third report. this is frustrating as i rely on a screen reader and cannot see. please role out the stable version of this program and fix any errors in the fcc editor with vallidation and not so strickt and be flexable. and also have a short cut to get back to the list of steps in the project. currently i have to go all the way back to the home page, then go to the list of the javascript section, then find the project, then scroll all the way down with my keyboard and screen reader. would be handy for a blind person, since i am the only one who is on the forum. please take my concerns seriously and do not fob me off. rant over.

**Comment by jeremylt** [comment]
Please create a post on the forum. The correct code for this step passes, so there is likely an issue with your full code.
